HunterPie.Core/Converters/ObservableConverter.cs

```diff
@@ -10,13 +10,20 @@ internal class ObservableConverter : JsonConverter
 
     public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
     {
-
         Type T = objectType.GetGenericArguments()[0];
-        object value = T.IsEnum
-            ? Enum.Parse(T, reader.Value.ToString())
-            : reader.Value != null
-                ? Convert.ChangeType(reader.Value, T)
+        object? value;
+        if (T.IsEnum)
+        {
+            string? stringValue = reader.Value?.ToString();
+            value = stringValue is not null && Enum.TryParse(T, stringValue, out object? enumValue)
```

Comment:
I think the value for enums should default to the first element in the enum and not `null`. Defaulting to `null` will cause another type of issue in the UI